CHANGELOG.md

```diff
@@ -9,6 +9,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Fixed
 
+- Fragile CMake logic for checking minimum version requirements for gFTL.
```

Comment:
Could you add this to a new:
```
## [2.12.1] - 2021-11-08
...
```
section above `[2.12.0]` and below `[Unreleased]`?

Reply:
will do